pkg/adapter/pods.go

```diff
@@ -343,7 +343,7 @@ func (r *LocalRuntime) CreatePod(ctx context.Context, cli *cliconfig.PodCreateVa
 			logrus.Debugf("Pod will use host networking")
 			options = append(options, libpod.WithPodHostNetwork())
 		case "":
-			return "", errors.Errorf("invalid value passed to --net: must provide a comma-separated list of CNI networks or host")
+			return "", errors.Errorf("invalid value passed to --network: must provide a comma-separated list of CNI networks or host")
```

Comment:
host is not mentioned  in the man page.  It should be removed here or added there.

Reply:
This belongs to ``CreatePod`` which includes host in its man page but I've corrected this in line 601.

I just wanted to correct ``--net`` to ``--network`` here.